src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarShowingNow.tsx

```diff
@@ -0,0 +1,72 @@
+import { Flex, Text, Spacer, Stack } from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+import { useTranslation } from "react-i18next"
+import { ArtworkSidebarShowingNow_artwork$data } from "__generated__/ArtworkSidebarShowingNow_artwork.graphql"
+import FairIcon from "@artsy/icons/FairIcon"
+import { RouterLink } from "System/Components/RouterLink"
+
+interface ArtworkSidebarShowingNowProps {
+  artwork: ArtworkSidebarShowingNow_artwork$data
+}
+
+export const ArtworkSidebarShowingNow: React.FC<ArtworkSidebarShowingNowProps> = ({
+  artwork,
+}) => {
+  const { t } = useTranslation()
+
+  const shouldRenderShowingNow = artwork.collectorSignals?.runningShow ?? false
+
+  if (!shouldRenderShowingNow) {
+    return null
+  }
+
+  const formatDate = (dateString: string) => {
+    return new Date(dateString).toLocaleDateString("en-US", {
+      month: "short",
+      day: "numeric",
+    })
+  }
+
+  const startAt = formatDate(
+    artwork.collectorSignals?.runningShow?.startAt ?? ""
+  )
+  const endAt = formatDate(artwork.collectorSignals?.runningShow?.endAt ?? "")
+
+  return (
+    <>
+      <Spacer y={4} />
```

Comment:
**Suggestion:** I noticed that there's a `Spacer` at the top and the bottom of the component, maybe we could add it to the `Flex` as margin Y?